src/main/java/tutorial/User.java

```diff
@@ -15,11 +22,14 @@ public class User {
      * @param id the User's id as assigned when the User is created
      */
 
-    public User(String displayName, int id) {
+    public User(String displayName, int id, String username, String password) {
         this.displayName = displayName;
-        this.id = id; //need to determine how we will be creating id and ensuring it does not already exist //
```

Comment:
can remove this line and the line below, we will be getting rid of id as a variable entirely